src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -183,7 +183,7 @@ SyntaxNodeAnalysisContext ctx
 				if( implicitType != null && implicitType.Kind != SymbolKind.ErrorType ) {
 
 					var baseExprType = implicitType.BaseType;
-					if( baseExprType.Equals( expressionType.OriginalDefinition, SymbolEqualityComparer.Default ) ) {
+					if( expressionType.OriginalDefinition.Equals( baseExprType, SymbolEqualityComparer.Default ) ) {
```

Comment:
test case?

Reply:
I can swap it back after my current build and try to figure out the syntax it was erroring on

Reply:
Could be its own PR too, feel free to merge and release this if you want.

Reply:
I'm getting a `NotImplementedException` during full_all but not VS anyway, so, have to figure that out